src/executor.rs

```diff
@@ -304,6 +304,7 @@ where
         callbacks: H,
         always_validate: bool,
         exits: Vec<u64>,
+        infinite_loop: bool,
```

Comment:
maybe just `cache_tb`, the name is a bit misleading

Reply:
Fixed :)